packages/ui/src/components/Designer/RightSidebar/DetailView/index.tsx

```diff
@@ -61,13 +61,8 @@ const DetailView = (props: DetailViewProps) => {
 
   useEffect(() => {
     const values: any = { ...activeSchema };
-    // [position] Change the nested position object into a flat, as a three-column layout is difficult to implement
```

Comment:
I looked at this code again and decided I wanted to delete it, so I did.
It's no longer a three-column layout, but I don't think there's any problem with that.

![CleanShot 2024-08-09 at 20 29 55@2x](https://github.com/user-attachments/assets/3d6ba9ea-243e-4575-ada1-58b02d3e9e42)